src/test/java/uk/gov/hmcts/reform/civil/launchdarkly/FeatureToggleServiceTest.java

```diff
@@ -195,6 +195,6 @@ void shouldCallBoolVariation_whenIsPbaV3EnabledInvoked() {
         givenToggle(pbaV3Key, true);
 
         assertThat(featureToggleService.isPbaV3Enabled()).isTrue();
-        verifyBoolVariationCalled(pbaV3Key, List.of("timestamp", "environment"));
+        //verifyBoolVariationCalled(pbaV3Key, List.of("timestamp", "environment"));
```

Comment:
This has been commented out for testing purposes, but must be uncommented before merge to master.

Reply:
I'll want an explanation on this, actually.

Maybe there's a better way.